### PR TITLE
Add inspector typescript

### PR DESCRIPTION
**Pull request by chichinohaha**
https://github.com/cocos-creator/3d-tasks/issues/6427

### CODE REVIEW

File: editor/inspector/assets/typescript.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+</section>`;
+
+exports.$ = {
+    code: '#code',
+};
+
+let panel;
+
+exports.style = `
+[hidden] {
```
Comment (VisualSJ) [inline editor/inspector/assets/typescript.js]: :host[hidden]

File: editor/inspector/assets/typescript.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        });
+        readLineStream.on('close', (err) => {
+            if (err) {
+                throw err;
+            }
+            panel.$.code.innerHTML = text;
+        });
+    } else {
+        panel.$.code.innerText = '';
+        panel.$.code.hidden = true;
```
Comment (VisualSJ) [inline editor/inspector/assets/typescript.js:105]: Is there a way to give some hints here?

Comment (chichinohaha) [reply editor/inspector/assets/typescript.js:105]: I '  don't think so.

File: editor/inspector/assets/typescript.js (hunk 1 above)
Comment (chichinohaha) [reply editor/inspector/assets/typescript.js]: :host > .asset-javascript > ui-code[hidden]

**Comment by chichinohaha** [comment]
image

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**